autoload/codefmt.vim

```diff
@@ -26,6 +26,7 @@
 " system.
 "
 " The current list of defaults by filetype is:
+"   * BUILD files (Bazel): buildifier
```

Comment:
Done. Personally, I think there should be exactly two lists:

1. On the Readme.
2. Available as :ListFormatTypes or tab-ing :FormatCode